make/go.mk

```diff
@@ -48,6 +39,9 @@ tidy:
 vet:
 	go vet ./...
 
+.PHONY: generate-assets
+generate-assets:
+
 .PHONY: pre-verify
-pre-verify: generate-assets
+pre-verify: 
```

Comment:
could we delete this target or will this cause issues with the verify-dependencies script ?

Reply:
This needs to be deleted from all the other repos, so i will do it as a separate PR, as it will require change to script too ..